src/frontends/pytorch/src/op/sort.cpp

```diff
@@ -41,6 +41,36 @@ OutputVector translate_sort(const NodeContext& context) {
     return topk->outputs();
 };
 
+OutputVector translate_sort_fx(const NodeContext& context) {
+    // aten.sort.default(Tensor self, int dim=-1, bool descending=False) -> (Tensor values, Tensor indices)
+    num_inputs_check(context, 1, 3);
+    const auto input_tensor = context.get_input(0);
+    bool descending = false;
+    int64_t dim = -1;
+
+    if (!context.input_is_none(1)) {
+        dim = context.const_input<int64_t>(1);
+    }
+    if (!context.input_is_none(1)) {
+        descending = context.const_input<bool>(2);
+    }
+
+    auto mode = descending ? ov::op::TopKMode::MAX : ov::op::TopKMode::MIN;
+    auto zero_axis = context.mark_node(opset11::Constant::create(element::i32, Shape{1}, {0}));
+    auto dim_axis = context.mark_node(opset11::Constant::create(element::i64, Shape{1}, {dim}));
+    auto shape = context.mark_node(std::make_shared<opset11::ShapeOf>(input_tensor));
+    auto k_values_node = context.mark_node(std::make_shared<opset11::Gather>(shape, dim_axis, zero_axis));
+    auto k_values = context.mark_node(std::make_shared<opset11::Squeeze>(k_values_node));
+    auto topk = context.mark_node(std::make_shared<opset11::TopK>(input_tensor,
+                                                                  k_values,
+                                                                  dim,
+                                                                  mode,
+                                                                  ov::op::TopKSortType::SORT_VALUES,
+                                                                  element::i64));
+    auto indices = context.mark_node(std::make_shared<ov::op::v0::Convert>(topk->output(1), element::i64));
```

Comment:
This part is similar between ts and fx, could you extract it in `common` function?